Xamarin.Forms.Xaml.UnitTests/ResourceDictionaryWithSource.xaml.cs

```diff
@@ -50,6 +50,8 @@ public void RelativeAndAbsoluteURI(bool useCompiledXaml)
 				Assert.That(((ResourceDictionary)layout.Resources["shortURI"])["sharedfoo"], Is.TypeOf<Style>());
 				Assert.That(((ResourceDictionary)layout.Resources["Colors"])["MediumGrayTextColor"], Is.TypeOf<Color>());
 				Assert.That(((ResourceDictionary)layout.Resources["CompiledColors"])["MediumGrayTextColor"], Is.TypeOf<Color>());
+				Assert.That(((ResourceDictionary)layout.Resources["BaseStyle"])["BaseLabelStyle"], Is.TypeOf<Style>());
+				Assert.That(((ResourceDictionary)layout.Resources["InheritedStyle"])[typeof(Label).FullName], Is.TypeOf<Style>());
```

Comment:
Do we have a test with multiple layers like this? Base -> Inherited -> Overrides? It probably will work, but a test to confirm and prevent future breaks will be great.